apps/courses/src/routes/users/dashboard/Mentor.tsx

```diff
@@ -352,27 +352,67 @@ export const MentorTabs = ({ courses, mentor }) => {
     );
   };
 
-  const MyActivity = () => {
-    const user: firebase.User = useUser();
-    const db = useFirestore();
-    const dbReviewsRef = db
+  const loadReviewData = async (db, uId, startAt, pageSize) => {
+    let dbRef = db
       .collection('users')
-      .doc(user.uid)
+      .doc(uId)
       .collection('reviews')
       // .where('status', '!=', 'pending')
       .orderBy('started_at', 'desc')
-      .limit(10);
-    const dbReviews: MentorReview[] = useFirestoreCollectionData(dbReviewsRef);
+      .limit(pageSize + 1);
```

Comment:
I see you're upping the pageSize by one here. Is this to keep track of the end of the queue?